sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample13_AuthenticateWithClientSecretCredential.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Identity;
+using Azure.Messaging.EventHubs.Samples.Infrastructure;
+
+namespace Azure.Messaging.EventHubs.Samples
+{
+    public class Sample13_AuthenticateWithClientSecretCredential : IEventHubsIdentitySample
+    {
+        /// <summary>
+        ///   The name of the sample.
+        /// </summary>
+        ///
+        public string Name { get; } = nameof(Sample13_AuthenticateWithClientSecretCredential);
+
+        /// <summary>
+        ///   A short description of the sample.
+        /// </summary>
+        ///
+        public string Description { get; } = "An example of authentication using ClientSecretCredential.";
```

Comment:
A bit more context may be helpful.  Perhaps something like:  `An example of creating an Event Hub client using a Azure Active Directory application with a client secret for authorization.` ?